### DISCUSSION TITLE
Archive a document by dropping in the sidebar

### DISCUSSION THREAD

**Discussion by iamsaumya** [outline/outline#2135] [Ideas]
While testing for bugs, I find myself with a lot of documents at times. Now, I either have to delete or archive the documents. Both the option are taxing for a large number of documents. 

From the user's point of view, they too will find it tedious to archive each document through the menu. 

How about dropping the document on the archive menu item in the sidebar? It will be much faster in this case. 

Considering the fact we already have drag and drop implemented in the app, the users are already accustomed to the notion of drag and drop.

**Comment by tommoor** [comment]
I think it's a good idea, we can do the same for trash too

**Reply by iamsaumya** [reply]
Is it possible to have both icons with lid open?

It will guide the user and will feel good if the icon change when document is hovered over Archive or Trash.

**Reply by tommoor** [reply]
Could be a cute detail! I wouldn't count it as a blocker though 😄

**Reply by tommoor** [reply]
Okay, I published `outline-icons@1.27.1` which has an `open` prop on `TrashIcon` and `ArchiveIcon` to trigger the open state 😉
